ProjectCoimbra.UWP/DataAccessLibrary/DataAccess.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed under the MIT License.
+
+namespace DataAccessLibrary
+{
+    using Microsoft.Data.Sqlite;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Windows.Storage;
+
+    /// <summary>
+    /// Data access class for the local User table in SQLite database
+    /// </summary>
+    public static class DataAccess
+    {
+        /// <summary>
+        /// Initializes the database, which is called when application starts
+        /// </summary>
+        public async static void InitializeDatabase()
+        {
+            await ApplicationData.Current.LocalFolder.CreateFileAsync("coimbra.db", CreationCollisionOption.OpenIfExists);
+            string dbpath = Path.Combine(ApplicationData.Current.LocalFolder.Path, "coimbra.db");
+            using (SqliteConnection db =
+               new SqliteConnection($"Filename={dbpath}"))
+            {
+                db.Open();
+
+                string tableCommand = "CREATE TABLE IF NOT " +
+                    "EXISTS User (Primary_Key INTEGER PRIMARY KEY, " +
+                    "Name NVARCHAR(2048) NULL)";
+
+                SqliteCommand createTable = new SqliteCommand(tableCommand, db);
+
+                createTable.ExecuteReader();
+            }
+        }
+
+        /// <summary>
+        /// Adds data to the User table
+        /// </summary>
+        /// <param name="inputText">The input text.</param>
+        public static void AddData(string inputText)
+        {
+            string dbpath = Path.Combine(ApplicationData.Current.LocalFolder.Path, "coimbra.db");
+            using (SqliteConnection db =
+              new SqliteConnection($"Filename={dbpath}"))
```

Comment:
The parameter to `SqliteConnection` is used in many places. We should save it in a `static` variable instead of recreating it each time.